recipe_scrapers/hilahcooking.py

```diff
@@ -0,0 +1,7 @@
+from ._abstract import AbstractScraper
+
+
+class HilahCooking(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "hilahcooking.com"
```

Comment:
```suggestion
        return "hilahcooking.com"
        
    def description(self):
        return self.soup.find("meta", {"itemprop": "description"}).get("content")

```

Since the Schema is returning a `description: ""` it can be gathered from elsewhere to ensure its descriptive.